modules/fundamental/src/sbom/model/details.rs

```diff
@@ -356,11 +349,7 @@ impl SbomAdvisory {
                 id: each.sbom_package.node_id.clone(),
```

Comment:
I think we can optimize this further by not returning the whole package info as I don't think it's useful in this context. That would also allow us to join less tables in the `relevant_advisory_info` query. But we can do that in a separate PR

Reply:
Yea, I agree, there's more potential. But I didn't want to go all crazy right before a release. :wink: